tests/components/motioneye/test_camera.py

```diff
@@ -231,7 +231,7 @@ async def test_get_still_image_from_camera(
 ) -> None:
     """Test getting a still image."""
 
-    image_handler = AsyncMock(return_value="")
+    image_handler = AsyncMock(return_value=web.Response(body=""))
```

Comment:
The mock was no longer sufficient since aiohttp now internally checked it was passed the correct type